docs/user/kafka_datastore.rst

```diff
@@ -300,6 +300,83 @@ Finally the Kafka Replay Consumer Feature Source can be queried:
 
     replayFeatureSource.getFeatures(timeFilter);
 
+Listening for Feature Events
+----------------------------
+
+The GeoTools API includes a mechanism to fire off a `FeatureEvent`_ object each time
+that there is an "event," which occur when data are added, changed, or deleted in a
```

Comment:
Nope, I think you're right. The first version was two sentences, where the second started "Events occur when data..." I forgot to change the verb tense when I put them together.